<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changecus</class>
 <widget class="QDialog" name="changecus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改顾客信息</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>年龄：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_age"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>231</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>住址：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="text_address"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_ok">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>130</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(勿以&quot;,&quot;结尾)</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>270</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>已购药品：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_gou"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>备注：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="text_mark"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>270</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>录入日期：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2019</year>
        <month>10</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>231</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>症状：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="text_zhengzhuang"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26pt &quot;方正舒体&quot;;</string>
   </property>
   <property name="text">
    <string>修改顾客信息</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>单号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_danhao"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="addcusLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>性别：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>男</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">sexGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>女</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">sexGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>200</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>经办人：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_sales"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pb_cancel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="addcusLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_name"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="addcusLayout_4">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>电话：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_phone"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>234</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="addcus_Layout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>编号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_num"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>获取信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sexGroup"/>
 </buttongroups>
</ui>
